scripts/patch-realsense-ubuntu-lts-hwe.sh

```diff
@@ -93,7 +93,8 @@ k_tick=$(echo ${kernel_version[2]} | awk -F'-' '{print $2}')
 [ $k_maj_min -eq 504 ] && [ $k_tick -ge 156 ] && apply_hid_gyro_patch=1 && skip_hid_patch=1
 # For kernel versions 6+ powerline frequency already applied
 [ $k_maj_min -ge 600 ] && skip_plf_patch=1
-[ $k_maj_min -ge 605 ] && skip_md_patch=1
+# do not skip md patch - new d421.
+#[ $k_maj_min -ge 605 ] && skip_md_patch=1
```

Comment:
What will happen when we try to apply it on SKU's we already have the PID up streamed?
Will the patch fail?

Reply:
6.5 patch append only d421

Reply:
Oh I think I understand now,
Let's run and verify tomorrow so we can merge it.

Reply:
Verified with D421 camera?